master/buildbot/test/unit/test_reporter_gerrit.py

```diff
@@ -44,11 +44,26 @@ def testReviewCB(builderName, build, result, status, arg):
                             (GERRIT_LABEL_VERIFIED, verified))
 
 
+@defer.inlineCallbacks
+def testReviewCBDeferred(builderName, build, result, status, arg):
+    verified = 1 if result == SUCCESS else -1
+    yield
+    defer.returnValue(makeReviewResult(str({'name': builderName, 'result': result}),
+                                      (GERRIT_LABEL_VERIFIED, verified)))
+
+
 def testStartCB(builderName, build, arg):
     return makeReviewResult(str({'name': builderName}),
                             (GERRIT_LABEL_REVIEWED, 0))
 
 
+@defer.inlineCallbacks
+def testStartCBDeferred(builderName, build, arg):
+    yield
```

Comment:
@tardyp - thanks for bearing with me while I learn the twisted framework. 

I thought yielding nothing was "good enough" for a test - it showed we were capable of calling a deferred if necessary. Your proposal makes much more sense, though! Fixed.